app/views/sessions/login.html.erb

```diff
@@ -2,21 +2,60 @@
 
   <h1>Log In</h1>
 
-  <p><i>Use your <a href="http://publiclab.org" target="_blank">PublicLab.org</a> username to log in. If you do not yet have one, <a href="http://publiclab.org/signup">sign up there</a> first!</i></p>
+  <p style="font-family:'Junction Light','lucida grande','lucida sans console',sans-serif;font-size: 15px;">
+    Login with your <a href="http://publiclab.org" target="_blank">PublicLab.org</a> username
+  </p>
+  <div class='hr-or' style="font-size: 10px;"></div>
+  <p style="font-family:'Junction Light','lucida grande','lucida sans console',sans-serif;margin-left: 40px;font-size: 15px;">
+    Create a <a href="http://publiclab.org/signup">Public Lab account</a>
+  </p>
 
   <form id="open_id_form" method="GET" action="/session/new">
 
     <%= hidden_field_tag :authenticity_token, form_authenticity_token -%>
-    <%= hidden_field_tag :back_to,@referer  -%>
-
-    <input tabindex="1" placeholder="username" id="username" name="open_id" type="text" /></p>
-    <input type="hidden" name="return_to" value="<%= params[:return_to] %>" />
```

Comment:
May I know why we deleted line 10 and 13?
I think there is need of these lines in the controller

Reply:
Can you please ensure no params key is deleted from the views?

Reply:
params[:return_to] is changed to params[:back_to]